docs/pages/User/GettingStarted/test.md

```diff
@@ -9,8 +9,7 @@ This demonstrates the capability of Liqo to start a pod either in the local (*ho
 
 ## Start an Hello World pod
 
-To schedule a Pod in a foreign cluster, first you have to set the `KUBECONFIG` variable in the home cluster, such as in the following:
-
+First, ensure you are operating in your home cluster. Otherwise, set the `KUBECONFIG` variable to the correct value:
```

Comment:
```suggestion
This step requires some to issue some command to Kubernetes from a shell terminal. If your terminal runs in a machine that does not belong to your cluster (e.g., your personal laptop), you have to set the `KUBECONFIG` variable to the correct value:
```
==> I'm not sure the content of the above text is correct. It seems to me that the kubeconfig is needed anyway.

Reply:
The point of the first statement is to point to the Home API Server and not the foreign cluster we peered with. I try to rephrase it.